scripts/generate-google3-docs/resource-reference/templates/compute_computemanagedsslcertificate.tmpl

```diff
@@ -0,0 +1,53 @@
+{{template "headercomment.tmpl" .}}
+
+{% extends "config-connector/_base.html" %}
+
+{% block page_title %}{{ .Kind}}{% endblock %}
+{% block body %}
```

Comment:
Just one small issue: Could you add this line back to the template: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/8f61a4aba1735391d4a3331a0d593a74ee7f0a24/scripts/generate-google3-docs/resource-reference/templates/compute_computenetwork.tmpl#L7? Let's not remove common sections in the template file.